homework3/semi-uber/UI/src/app/regestration-page/regestration-page.component.ts

```diff
@@ -12,7 +15,11 @@ export class RegestrationPageComponent implements OnInit {
   passwordRegexp = "(?:[A-Z]+)(?:[a-z]+)(?:[0-9]+)(?:[@!#$%^&*()]+)";
   loginRegexp = ".{5,10}";
 
+  user: RegistrationModel;
+  formField = false;
+
   userData = new FormGroup({
```

Comment:
this should be done inside onInit hook

Reply:
I need user out site of init, because of i'm gone send it value at server.
formField - done

Reply:
You can create a public field `userData: FormGroup`  and the populate it in onInit. That's not really readable to put such kind of logic to the top of the class.